src/main/java/com/dampcake/bencode/BencodeOutputStream.java

```diff
@@ -126,13 +126,13 @@ public void writeDictionary(final Map<?, ?> m) throws IOException {
     private static String encode(final String s) {
         if (s == null) throw new NullPointerException("s cannot be null");
 
-        return String.format("%d%s%s", s.length(), Bencode.SEPARATOR, s);
+        return String.format("%s%s%s", Integer.toString(s.length()), Bencode.SEPARATOR, s);
```

Comment:
What does this do? When would `s.legnth()` ever return anything that would need to be toString'd?

Reply:
`String.format("%d", ...)` uses locale digits.
`Integer.toString(...)` uses "normal" digits.
Or locale may be set on method calls: `String.format(Locale.ENGLISH, "%d", ...)`.

Reply:
Makes sense. I think I would rather use the Locale.ENGLISH solution as that looks to be the more accepted solution. Also if you could update the BencodeOutputStreamTest to run parameterized with all the locales that would be awesome!